clients/client-java/src/main/java/org/apache/gravitino/client/FilesetCatalog.java

```diff
@@ -238,7 +241,23 @@ public boolean dropFileset(NameIdentifier ident) {
   @Override
   public String getFileLocation(NameIdentifier ident, String subPath)
       throws NoSuchFilesetException {
-    throw new UnsupportedOperationException("Not implemented");
+    checkFilesetNameIdentifier(ident);
+    Namespace fullNamespace = getFilesetFullNamespace(ident.namespace());
+
+    CallerContext callerContext = CallerContext.CallerContextHolder.get();
+
+    Map<String, String> params = new HashMap<>();
+    params.put("subPath", RESTUtils.encodeString(subPath));
```

Comment:
I would suggest we don't use camel style for param here "subPath" and path "fileLocation", how about "sub_path" and "location"?

Reply:
Fixed